webcompat/static/js/lib/bugform.js

```diff
@@ -227,6 +217,20 @@ function BugForm() {
     }
   };
 
+  this.clearStepsToReproducePlaceholder = function() {
+    var stepsToReproduceStrAr = this.stepsToReproduceField
+       .val()
+       .split("4. Nothing happened.");
```

Comment:
(it's entirely possible a user types in their own reply and it happens to have a step 4 that says "Nothing happened", so we'll end up doing the wrong thing.)